Sources/Certificate Pinning/CertificateHasher.swift

```diff
@@ -53,10 +53,10 @@ struct CertificateHasher {
     }
     
     static func checkValidity(of trust: SecTrust) -> Bool {
-        var result = SecTrustResultType.invalid
-        let status = SecTrustEvaluate(trust, &result)
+        var result: CFError?
+        let status = SecTrustEvaluateWithError(trust, &result)
         
-        return (status == errSecSuccess) ? (result == .unspecified || result == .proceed) : false
+        return status || result == nil
```

Comment:
I was catching up on some old PRs today and made a comment in the relevant PR, but is there a technical reason why the logic here changed?

Reply:
Yea, SecTrustEvalute() was deprecated.

Reply:
Thanks, I just saw Daniel's comment about it as well.